packages/mds-compliance/tests/api.spec.ts

```diff
@@ -526,7 +526,7 @@ describe('Tests Compliance API:', () => {
   describe('Time Violation Test: ', () => {
     before(done => {
       const devices: Device[] = makeDevices(15, now())
-      const events = makeEventsWithTelemetry(devices, now() - 1000000, CITY_OF_LA, 'trip_end')
+      const events = makeEventsWithTelemetry(devices, now() - 21 * 60 * 1000, CITY_OF_LA, 'trip_end')
```

Comment:
There are helpers in `mds-utils` where this could be `now() - mins(21)` for example.